build/en/index.html

```diff
@@ -0,0 +1,104 @@
+            <li> Nodes can be &quot;anything&quot; (e.g., text, images, XML records)</li>
+            <li> Edges can hold arbitrary data (e.g., weights, time-series)</li> 
+            <li> Open source <a href="https://raw.githubusercontent.com/networkx/networkx/master/LICENSE.txt">3-clause BSD license</a> </li>
+            <li> Well tested with over 90% code coverage</li>
+            <li> Additional benefits from Python include fast prototyping, easy to teach, and multi-platform</li>
+            </ul>
+
+          </div>
+        </div>
+      </div>
+
+      <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
+        <div class="sphinxsidebarwrapper">
+          <div class="sidebarblock">
+          <h2>Contact</h2>
+            <ul class="title">
+              <li><a class="reference external" href="http://groups.google.com/group/networkx-discuss/"> Mailing list</a></li>
+              <li><a class="reference external" href="https://github.com/networkx/networkx/issues"> Issue tracker</a></li>
+              <li><a class="reference external" href="https://github.com/networkx/networkx"> Source</a></li> 
+            </ul>
+          </div>
+          
+          <div class="sidebarblock">
+          <h2>Releases</h2>
+          
+            <div class="tile">
+            <h4>Stable (<a href="https://networkx.org/documentation/stable/release/release_3.4.2.html">notes</a>)</h4>
+              <a href="http://pypi.python.org/pypi/networkx">3.4.2 — October 2024</a><br/>
+              <a href="https://networkx.org/documentation/stable/"><b>Documentation</b></a>
+            </div></br>
+            
+            <div class="tile">
+            <h4>Latest (<a href="https://networkx.org/documentation/latest/release/release_dev.html">notes</a>)</h4>
+              <a href="https://github.com/networkx/networkx">3.5 development</a><br/>
+              <a href="https://networkx.org/documentation/latest/"><b>Documentation</b></a>
+            </div></br>
+          
+            <div class="tile">
+            <h4><a id="other">Archive</a></h4>
+            
+              <ul id="versionList" style="display: none;">
+                  <script src="../_static/docversions.js"></script>
+                  <script type="text/javascript">
+                      insert_version_links();
+                  </script>
+              </ul>
+            
+              <script type="text/javascript">
+                  $("#other").click(function() {
+              	$("#versionList").toggle();
+              });
+              </script>
+            </div></br>
+          
+            <div class="tile">
+              <h4>Language</h4>
+              <select id="language-switcher" onchange="change_language()">
+              </select>
+            </div></br>
+          </div>
+        </div>
+      </div>
+      <div class="clearer"></div>
+    </div>
+
+  <div class="footer">
+    &copy;2014-2025, NetworkX developers.
```

Comment:
@melissawm 

We could add something like this to have the date update automatically to answer one of the questions @Schefflera-Arboricola had

`&copy;2014-<span id="copyright-year"><span>, NetworkX developers.`

and then add some javascript that does something like 
```js
    document.addEventListener("DOMContentLoaded", function() {
      const yearSpan = document.getElementById("copyright-year");
      const currentYear = new Date().getFullYear();
      yearSpan.textContent = currentYear;
    });
```

Reply:
I'll address it in my next commit, thanks!